tests/cli/test.cli.js

```diff
@@ -0,0 +1,41 @@
+import {expect} from 'chai';
+import {spy} from 'sinon';
+
+import {Program} from '../../src/cli';
+
+
+describe('cli.Program', () => {
+
+  it('executes a command callback', () => {
+    let thing = spy(() => new Promise((resolve) => resolve()));
+    let program = new Program(['thing'])
+      .command('thing', 'does a thing', thing);
+    return program.run()
+      .then(() => {
+        expect(thing.called).to.be.equal(true);
```

Comment:
Was there a rationale to use expect style here over the TDD style used elsewhere e.g. addons-linter or is this what was used on JPM?

The thing I always like about the standard assert style is it's super direct and obvious.